test/e2e/system/system_utils.go

```diff
@@ -20,9 +20,9 @@ import (
 	"strings"
 )
 
-// IsMasterNode returns true if given node is a registered master.
-// TODO: find a better way of figuring out if given node is a registered master.
-func IsMasterNode(nodeName string) bool {
+// DeprecatedIsMasterNode returns true if given node is a registered master.
+// DEPRECATED: use a label selector provided by test initialization.
+func DeprecatedIsMasterNode(nodeName string) bool {
```

Comment:
discourage even more? `DeprecatedMightBeMasterNode`? also add a comment explicitly saying this should not be updated to use the node role label

Reply:
OK